airbyte-integrations/connectors/source-postgres/src/main/resources/spec.json

```diff
@@ -149,10 +149,12 @@
               },
               "ca_certificate": {
                 "type": "string",
-                "title": "CA certificate",
+                "title": "CA Certificate",
```

Comment:
Capitalized `Certificate` in this title to make it match the title of this field in the Password authentication case